client/python/modeldb/utils/ConfigUtils.py

```diff
@@ -1,68 +1,71 @@
 import sys
-import yaml
+import json
 import ConfigConstants as constants
 
 class ConfigReader(object):
     def __init__(self, filename):
         # TODO: need to deal with errors here
-        self.config =  yaml.load(file(filename, 'r'))
+        self.config =  json.load(file(filename, 'r'))
         self.validate_config()
 
     def validate_config(self):
         # TODO: what all do we expect to be here?
-        if constants.PROJECT_KEY not in self.config:
+        if 'project' not in self.config:
             print "Project must be defined in config file. Cannot proceed."
             sys.exit(-1)
-        if constants.GIT_USERNAME_KEY not in self.config:
```

Comment:
I don't quite get the reason for using the strings in this file but using constants.ABC in other files. I see that this is what you were asking about before -- the reason I had pulled out the constants was in case we wanted to change one of them to another string. Changing them in one place is a lot easier than changing them in multiple files. I'd actually recommend sticking with one or another. 

Not sure if I'm missing something here to please let me know if I am!